test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -45,6 +48,57 @@ public void AutoFitMultiTest()
             Assert.Equal(NumberDataViewType.Single, scoredData.Schema[DefaultColumnNames.PredictedLabel].Type);
         }
 
+        [TensorFlowFact]
+        public void AutoFitImageClassificationTrainTest()
+        {
+            var context = new MLContext();
+            var datasetPath = DatasetUtil.GetFlowersDataset();
+            var columnInference = context.Auto().InferColumns(datasetPath, "Label");
+            var textLoader = context.Data.CreateTextLoader(columnInference.TextLoaderOptions);
+            var trainData = context.Data.ShuffleRows(textLoader.Load(datasetPath), seed: 1);
+            var originalColumnNames = trainData.Schema.Select(c => c.Name);
+            TrainTestData trainTestData = context.Data.TrainTestSplit(trainData, testFraction: 0.2, seed: 1);
+            IDataView trainDataset = SplitUtil.DropAllColumnsExcept(context, trainTestData.TrainSet, originalColumnNames);
+            IDataView testDataset = SplitUtil.DropAllColumnsExcept(context, trainTestData.TestSet, originalColumnNames);
+            var result = context.Auto()
+                            .CreateMulticlassClassificationExperiment(0)
+                            .Execute(trainDataset, testDataset, columnInference.ColumnInformation);
+
+            //Known issue, where on Ubuntu there is degradation in accuracy.
```

Comment:
Can you link to the issue here? This seems like something that needs to be addressed.

Reply:
Related comment -- https://github.com/dotnet/machinelearning/pull/4395#discussion_r339356839

Reply:
@eerhardt There is no issue for this because we haven;t fully investigated the cause or even understand the problem, right now my suspicion is bad images format for Linux since it only seems to happen on the smaller version of flowers dataset that has 50 images. It seems there are two images that cause this kind of drop. I will have someone investigate this later.

Reply:
Can you log an issue for this? That is how work gets tracked. Just because it hasn't been fully investigated doesn't mean there shouldn't be an issue logged.